src/components/App/index.tsx

```diff
@@ -1,47 +1,48 @@
-import React from "react"
+// import React from "react"
 import "./styles.css"
+import { AppHeader } from "../AppHeader"
+import { ButtonCreateTask } from "../ButtonCreateTask"
+import { FormNewTask } from "../FormNewTask"
+import { TaskCard } from "../TaskCard"
+import { TaskCounter } from "../TaskCounter"
+
+export type Task = { task: string }
 
 export function App() {
-  const [count, setCount] = React.useState(0)
-  const changeCount = () => {
-    setCount(1)
-  }
+  // const [modal, setModal] = React.useState(false)
+  // const handleModal = (e) => {
+  //   setModal(true)
+  // }
+  // const [taskArray, setTasksArray] = React.useState([])
+  // const addTask = (e) => {
+  //   setTasksArray([...taskArray, e.target.value])
+  // }
 
+  const tasks: Task[] = [{ task: "finish the app" }, { task: "do a dance" }]
+  const completedTasks: Task[] = [
+    { task: "make an array" },
+    { task: "do it live" },
+  ]
   return (
     <div className="App">
-      <header className="App-header">
-        <h1>My Tasks</h1>
-      </header>
-      <body>
-        <h3>Active Tasks</h3>
-        <form className="newTaskForm">
-          <label>New Task</label>
-          <input></input>
-          <button>Cancel</button>
-          <button>Save</button>
-        </form>
-        <section className="TaskList">
-          <div className="taskCard">
-            <label>
-              <input name="task" type="checkbox" />
-              Complete development of apprenticeship practice to-do app
-            </label>
-            <button>Edit</button>
-            <button>Delete</button>
-          </div>
-        </section>
-        <button className="newTaskButton" onClick={changeCount}>
-          Create New Task
-        </button>
-        <section className="CompleteList">
-          <h3>Completed Tasks</h3>
-          <p>Count: {count}</p>
-          <label>
-            <input name="task" type="checkbox" />
-            Lorem ipsum
-          </label>
-        </section>
-      </body>
+      <AppHeader />
+      <h3>Active Tasks</h3>
+      <div className="TaskList">
+        {" "}
+        {tasks.map((taskItem: Task) => {
+          return <TaskCard task={taskItem} />
```

Comment:
I think you added indexes here (in what you demoed today) 💯 For now it's totally fine to use the index as the key (since we don't have any task ids), but here are some interesting articles on why/when to not use the index as a key. The second one also has an example of creating your own id (without a library) like we talked about today.

https://reactjs.org/docs/lists-and-keys.html#keys
https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318